topics/puppeteer/index.md

```diff
@@ -0,0 +1,8 @@
+---
+display_name: Puppeteer
+short_description: Headless Chrome Node.js API
+topic: puppeteer
+related: nodejs, API, headless-browsers
+wikipedia_url: https://en.wikipedia.org/wiki/Chromium_(web_browser)
```

Comment:
Linking to the Chromium wikipedia page doesn't seem like the right thing to do here. This topic isn't for Chromium itself, it's for Puppeteer.

Is there a top-level project homepage we could link to instead (with a `url:` key instead of `wikipedia_url:`)?

Reply:
@smashwilson Made the necessary changes